testsuite/data/src/test/resources/io/smallrye/openapi/testdata/components.schemas.kotlin-value-class-propname.json

```diff
@@ -5,7 +5,9 @@
       "KotlinBean": {
         "required": [
           "description",
-          "nestedCollection"
+          "nestedCollection",
+          "actualCustomName",
+          "requiredValue"
         ],
```

Comment:
Given that `description`, `nestedCollection` and `customName` are required without an annotation, does the `@Required` annotation on `requiredValue` actually do anything here?

Reply:
Good eye! You're right. Because of the generated `@NotNull` annotation for a non-nullable value, and the lack of processing of default values for `@Serializable` classes, in this case the `@Required` annotation actually doesn't change the outcome of the schema.

I have replaced this property with a `requiredNullableValue` property instead, where the `@Required` annotation correctly alters the required-ness of the property (that would otherwise be optional).